test/test_helper.rb

```diff
@@ -3,6 +3,7 @@
 require 'active_support/core_ext/kernel/reporting'
 require 'active_model'
 require 'rails'
+require 'mongoid'
```

Comment:
This `require` causes mongomapper tests failure on Travis. Can you please wrap it in `begin ... rescue LoadError`?